src/module/features/chat-alert.ts

```diff
@@ -242,20 +242,53 @@ const ITEM_TYPE_HANDLERS: { [key: string]: ItemTypeHandler } = {
     if (data.data?.rank) {
       const oldRank = game.i18n.localize(RANKS[progressData.data.rank])
       const newRank = game.i18n.localize(RANKS[data.data.rank])
-      return game.i18n.format('IRONSWORN.ChangeLog.rankChanged', {
+      return game.i18n.format('IRONSWORN.ChatAlert.rankChanged', {
         old: oldRank,
         new: newRank,
       })
     }
     if (data.data?.current !== undefined) {
       const advanced = data.data.current > progressData.data.current
       return game.i18n.localize(
-        `IRONSWORN.ChangeLog.progress${advanced ? 'Advanced' : 'Reduced'}`
+        `IRONSWORN.ChatAlert.progress${advanced ? 'Advanced' : 'Reduced'}`
       )
     }
+    if (data.data?.clockTicks !== undefined) {
+      const change = data.data.clockTicks - progressData.data.clockTicks
+      const advanced = data.data.clockTicks > progressData.data.clockTicks
+      const completed = data.data.clockTicks >= progressData.data.clockMax
+      let i18nKey = 'IRONSWORN.ChatAlert.clock'
+      switch (true) {
+        case completed: {
+          i18nKey += 'Completed'
+          break
+        }
+        case advanced: {
+          i18nKey += 'Advanced'
+          break
+        }
+        default: {
+          i18nKey += 'Set'
+          break
+        }
+      }
+      if (change > 2) {
+        i18nKey += 'MultipleSegments'
```

Comment:
This doesn't quite agree with `en.json`; there it's `ManySegments`.

Reply:
> This doesn't quite agree with `en.json`; there it's `ManySegments`.

whoops, that's what i get for renaming them from github's web interface because i don't want to fire up vscode, haha